backend/api/Services/SourceService.cs

```diff
@@ -0,0 +1,96 @@
+﻿
+
+using System.Text.Json;
+using Api.Database.Models;
+using Api.Options;
+using Azure.Storage.Blobs;
+using Microsoft.Extensions.Options;
+
+namespace Api.Services
+{
+
+    public interface ISourceService
```

Comment:
Suggestion is to remove all functionality related to managing blobs from this file and use what is implemented in #865 instead to have it more modular.

Additionally, `SourceService` is currently only interacting with `CustomMission`. I would rename the entire service to `CustomMissionService`.

Reply:
Will not review this file further as I expect it will change a lot because of this.

Reply:
I will implement this change last, as it requires rebasing the branch from main